src/lib/web3/rainbowkit/provider.tsx

```diff
@@ -0,0 +1,25 @@
+import '@rainbow-me/rainbowkit/styles.css';
+
+import { ReactNode } from 'react';
+import { WagmiProvider } from 'wagmi';
+import { getWagmiConfig } from '../wagmi-config';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { RainbowKitProvider as RKProvider, darkTheme } from '@rainbow-me/rainbowkit';
+
+const queryClient = new QueryClient();
+
+export interface RainbowKitProviderProps {
+  children: ReactNode;
+}
+
+export const RainbowKitProvider = ({ children }) => {
+  return (
+    <WagmiProvider config={getWagmiConfig()}>
+      <QueryClientProvider client={queryClient}>
```

Comment:
[`react-query` provider is required by RainbowKit](https://www.rainbowkit.com/docs/installation#wrap-providers)